components/molecule/coachmark/src/Tooltip/index.js

```diff
@@ -126,10 +125,18 @@ const Tooltip = ({
 }
 
 Tooltip.propTypes = {
-  defaultTooltipOptions: PropTypes.shape({
```

Comment:
🍆 is this a breaking change? 🤔

Reply:
It is in V.0 so it is supposed no one is still using it